<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mind Your OSCs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="emotivIPedit">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="emotivPortEdit">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>220</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="oscIPedit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="oscPortEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>220</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>7400</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="emotivCombo">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>150</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Control Panel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EmoComposer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Emotiv Engine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="leftLine">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>20</y>
      <width>20</width>
      <height>507</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="rightLine">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>-7</y>
      <width>40</width>
      <height>553</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="emotivLabel">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>107</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Emotiv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="emotivConnectButton">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="emotivDisconnectButton">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oscConnectButton">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>271</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oscDisconnectButton">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="emotivConnectionStatusLabel">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>114</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cogLabel">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>69</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Cognitiv</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cogTypeLine">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>104</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="cogPowerLCD">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>132</y>
      <width>64</width>
      <height>27</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="cogPowerLabel">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>134</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Power:</string>
    </property>
   </widget>
   <widget class="QLabel" name="affLabel">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>17</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Affectiv</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>45</y>
      <width>178</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="affEngageBoredLabel">
       <property name="text">
        <string>Engagement/Boredom</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="affFrustrationLabel">
       <property name="text">
        <string>Frustration</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="affMeditationLabel">
       <property name="text">
        <string>Meditation</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="affExciteLabel">
       <property name="text">
        <string>Excitement</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="affExcitementLTLabel">
       <property name="text">
        <string>Excitement Long Term</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="affEngageBoredLCD">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="affFrustrationLCD">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="affMeditationLCD">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="affExcitementLCD">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLCDNumber" name="affExcitementLTLCD">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>196</y>
      <width>389</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>386</x>
      <y>1</y>
      <width>40</width>
      <height>202</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="affLabelFrame">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>14</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="cogLabelFrame">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>66</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="emotivLabelFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="oscLabelFrame">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>99</y>
      <width>55</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="oscLabel">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>6</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>OSC</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>424</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="expSmileLabel">
       <property name="text">
        <string>Smile</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="expClenchLabel">
       <property name="text">
        <string>Clench</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="expSmirkRightLabel">
       <property name="text">
        <string>Smirk Right</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="expLaughLabel">
       <property name="text">
        <string>Laugh</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="expSmirkLeftLabel">
       <property name="text">
        <string>Smirk Left</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="expSmileLCD"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="expClenchLCD"/>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="expSmirkLeftLCD"/>
     </item>
     <item row="1" column="4">
      <widget class="QLCDNumber" name="expSmirkRightLCD"/>
     </item>
     <item row="1" column="5">
      <widget class="QLCDNumber" name="expLaughLCD"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>350</y>
      <width>161</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="expRaiseBrowLabel">
       <property name="text">
        <string>Raise Brow</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="expRaiseBrowLCD">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="expFurrowBrowLabel">
       <property name="text">
        <string>Furrow Brow</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="expFurrowBrowLCD">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>418</y>
      <width>371</width>
      <height>92</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>14</y>
       <width>46</width>
       <height>26</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>16</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>12</y>
       <width>54</width>
       <height>27</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_7">
     <property name="geometry">
      <rect>
       <x>228</x>
       <y>15</y>
       <width>58</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>308</x>
       <y>18</y>
       <width>47</width>
       <height>17</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>209</y>
      <width>176</width>
      <height>112</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="expWinkLeftCheck">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="expLookLeftCheck">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="expWinkRightCheck">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="expLookRightCheck">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="expWinkRightLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wink Right</string>
     </property>
    </widget>
    <widget class="QLabel" name="expLookRightLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Look Right</string>
     </property>
    </widget>
    <widget class="QLabel" name="expLookLeftLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Look Left</string>
     </property>
    </widget>
    <widget class="QLabel" name="expWinkLeftLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Wink Left</string>
     </property>
    </widget>
    <widget class="QLabel" name="expBlinkLabel">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>35</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Blink</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="expBlinkCheck">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>36</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QFrame" name="expLabelFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="expLabel">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>5</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Expressiv</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>344</y>
      <width>176</width>
      <height>65</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="OSCConnectedLabel">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>144</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cmdLnEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>false</bool>
  </property>
  <property name="gridSnapY">
   <bool>false</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
